### PR TITLE
Update GoogleTests.java

### PR DESCRIPTION
**Pull request by AleksandrZhuravel**
1. Selectors "q" and "html" replaced by class selectors.

### CODE REVIEW

**Comment by AleksandrZhuravel** [comment]
1. Selectors "q" and "html" replaced by class selectors.

File: src/test/java/tests/GoogleTests.java [hunk 1]
```diff
@@ -14,9 +14,9 @@ void selenideSearchTest() {
         open("https://google.com");
 
         // Ввести Selenide в поиск
-        $(byName("q")).setValue("Selenide").pressEnter();
+        $(".gLFyf").setValue("Selenide").pressEnter();
```
Comment (kristina71) [inline src/test/java/tests/GoogleTests.java:17]: `$(".gLFyf")` Perhaps, This is a dynamic class that can be generated when rendering a page. Also, this locator is intuitively incomprehensible when reading code. It is better not to get attached to it.

File: src/test/java/tests/GoogleTests.java [hunk 2]
```diff
@@ -14,9 +14,9 @@ void selenideSearchTest() {
...
 
         // Проверить, что Selenide появился в результатах поиска
-        $("html").shouldHave(text("ru.selenide.org"));
+        $(".iUh30").shouldHave(text("ru.selenide.org"));
```
Comment (kristina71) [inline src/test/java/tests/GoogleTests.java:20]: the same problem

**Review by kristina71 — CHANGES_REQUESTED** [review]
And also I don't see the Yandex test. You need to create one more test (https://qa.guru/pl/teach/control/lesson/view?id=161420799&editMode=0 - step 3)

File: src/test/java/tests/GoogleTests.java [hunk 3]
```diff
@@ -14,9 +14,22 @@ void selenideSearchTest() {
...
 
         // Проверить, что Selenide появился в результатах поиска
-        $(".iUh30").shouldHave(text("ru.selenide.org"));
+        $("html").shouldHave(text("ru.selenide.org"));
     }
+    
+     @Test
+    void selenideSearchYandexTest() {
+        // Открыть google
+        open("https://google.com");
```
Comment (kristina71) [inline src/test/java/tests/GoogleTests.java:26]: Why do you open google.com? You should open yandex.ru and create any test

For example:
`@Test
    void emptySearch(){
        open("https://yandex.ru");
        $(".input__control").setValue("").pressEnter();
        $(".misspell__message").shouldHave(text("Задан пустой поисковый запрос"));
    }`

File: src/test/java/tests/GoogleTests.java [hunk 4]
```diff
@@ -14,9 +14,22 @@ void selenideSearchTest() {
...
+
+        // Ввести Selenide в поиск
+        $(byName("q")).setValue("Selenide").pressEnter();
+
+        // Проверить, что Selenide появился в результатах поиска
+        $("html").shouldHave(text("ru.selenide.org"));
```
Comment (kristina71) [inline src/test/java/tests/GoogleTests.java:32]: What is the difference between this test and the previous one?

**Review by kristina71 — CHANGES_REQUESTED** [review]

**Comment by AleksandrZhuravel** [comment]
You were right about my request. I reversed my false changes and added new Yandex-test. 
I also reviewed the code in accordance with the wishes of Stanislav Vasenkov and added the UTF-8 encoding to the file "build.gradle".

File: src/test/java/tests/GoogleTests.java [hunk 5]
```diff
@@ -10,26 +10,24 @@
...
+
+
+        $(".input__control").setValue("Selenide").pressEnter();// Ввести Selenide в поиск
 
-        // Ввести Selenide в поиск
-        $(byName("q")).setValue("Selenide").pressEnter();
 
-        // Проверить, что Selenide появился в результатах поиска
-        $("html").shouldHave(text("ru.selenide.org"));
+        $("html").shouldHave(text("ru.selenide.org"));// Проверить, что Selenide появился в результатах поиска
```
Comment (kristina71) [inline src/test/java/tests/GoogleTests.java:30]: better use specific locator like ".organic__subtitle b"

**Review by kristina71 — CHANGES_REQUESTED** [review]

**Review by kristina71 — APPROVED** [review]

**Comment by AleksandrZhuravel** [comment]
I changed the locator to a locator by class (different from yours) and renamed Java class. You should rename GoogleTests.java according to the Java class name.

**Pull request merged**